src/transformations.js

```diff
@@ -0,0 +1,7 @@
+import { Class, Constructor, Node, File, traverse } from './model'
+
+export const addDefaultConstructor = traverse({
+  [File]: ({ content }) => File(...content.map(addDefaultConstructor)),
+  [Class]: (node) => (node.members.some(c => c.is(Constructor)) ? node : node.copy({ members: ms => [Constructor()()(), ...ms] })),
+  [Node]: (node) => node
+})
```

Comment:
Yes. It is a bit of an overkill for this in particular but it's a useful operation for general transforming.